packages/sanity/src/structure/panes/documentList/SheetListCell.tsx

```diff
@@ -0,0 +1,58 @@
+/* eslint-disable i18next/no-literal-string */
+import {Select, TextInput} from '@sanity/ui'
+import {type CellContext} from '@tanstack/react-table'
+import {type FormEvent, useCallback, useEffect, useState} from 'react'
+import {type SanityDocument} from 'sanity'
+
+interface SheetListCellProps extends CellContext<SanityDocument, unknown> {
+  type: any
```

Comment:
What is the type of `type`? Can you clarify it more?

Reply:
Resolved by more accurately typing the `schemaType` in `DocumentSheetListPane` with a type asserter for a document schema which means that the `schemaType` type elsewhere is now correct